wandb/sdk/wandb_run.py

```diff
@@ -3112,7 +3112,7 @@ def _prepare_artifact(
     def log_model(
         self,
         path: StrPath,
-        model_name: Optional[str] = None,
+        name: Optional[str] = None,
         aliases: Optional[List[str]] = None,
     ) -> None:
         """Declare a model artifact as an output of a run.
```

Comment:
```suggestion
        """Logs a model containing the contents inside the path to a run and marks it as output to this run. To use this method, ensure that there is an active run that already exists in this context.
```

Reply:
I disagree that we should add this: `To use this method, ensure that there is an active run that already exists in this context` as the method is on a run itself (as we discussed earlier).